### PR TITLE
fix: lint

### PR DESCRIPTION
**Pull request by koca**
Seems like we weren't checking linting at all 😊

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "build:dev": "yarn workspace @chakra-ui/vue run rollup --config rollup.dev.config.js",
     "build": "yarn workspace @chakra-ui/vue build",
     "dev": "yarn workspace @chakra-ui/vue dev",
-    "lint": "vue-cli-service lint",
+    "lint": "vue-cli-service lint packages/chakra-ui-core",
```
Comment (codebender828) [inline package.json:26]: How does this affect the other packages? We should probably change this to:
`packages/*` and then in the `.eslintignore` file, we can uncomment the `chakra-ui-theme` line.

Comment (koca) [reply package.json:26]: let me check

Comment (koca) [reply package.json:26]: i think each package should have its own `eslintrc` file or lint script. Not sure if we should be using `vue-cli-service` in root package.json tho.

Comment (koca) [reply package.json:26]: Lets get this merged and later we should refactor eslint and package.json(root).

Comment (codebender828) [reply package.json:26]: I see. This is good for now. We can do some research on how to improve this later.

**Pull request merged**